libconcurrent/concurrent/dsmqueue.c

```diff
@@ -38,8 +38,11 @@ inline static RetVal serialDequeue(void *state, ArgVal arg, int pid) {
     Node *node = (Node *)st->first;
 
     if (st->first->next != null) {
+        RetVal ret = node->val;
+
```

Comment:
Remove extra line as you did in other files

Reply:
done